// @northlight/tokens/dist/cjs/tokens.js
'use strict';

var borderRadius$2 = {
	button: {
		"default": "0.375rem",
		full: "999px"
	},
	checkbox: {
		"default": "0.125rem",
		full: "999px"
	},
	input: {
		outline: "0.375rem",
		filled: "0.375rem",
		flushed: "0rem"
	},
	select: {
		"default": "0.375rem"
	},
	textarea: {
		"default": "0.375rem"
	},
	tag: {
		"default": "999px"
	},
	toast: {
		"default": "0.375rem"
	},
	"input-stepper": {
		"default": "0.25rem"
	},
	scrollbar: {
		thumb: "0.5rem"
	},
	avatar: {
		rounded: "999px",
		square: "0.375rem"
	},
	modal: {
		dialog: "0.375rem"
	},
	tooltip: {
		"default": "0.25rem"
	},
	"flip-button": {
		"default": "0.375rem"
	},
	badge: {
		"default": "0.125rem"
	},
	statusBlock: {
		"default": "0.375rem"
	},
	"switch": {
		full: "999px"
	}
};
var borderWidth$1 = {
	radio: {
		"default": "0.0625rem",
		"checked-sm": "0.25rem",
		"checked-md": "0.25rem",
		"checked-lg": "0.375rem"
	},
	input: {
		"default": "0.0625rem"
	},
	select: {
		"default": "0.0625rem"
	},
	textarea: {
		"default": "0.0625rem"
	},
	spinner: {
		"default": "0.125rem"
	},
	modal: {
		header: "0.0625rem"
	},
	step: {
		"default": "0.125rem"
	},
	"flip-button-group": {
		"default": "0.125rem"
	},
	badge: {
		outlined: "0.0625rem"
	}
};
var color$2 = {
	background: {
		"default": "#ffffff",
		button: {
			"default": "#eaf3ff",
			"default-hover": "rgba(234, 243, 255, 0.5)",
			"default-active": "#eaf3ff",
			danger: "#fddcdb",
			"danger-hover": "rgba(253, 220, 219, 0.5)",
			"danger-active": "#fddcdb",
			success: "#def3e9",
			"success-hover": "rgba(222, 243, 233, 0.5)",
			"success-active": "#def3e9",
			brand: "#2a8bfb",
			"brand-hover": "rgba(42, 139, 251, 0.8)",
			"brand-active": "#2a8bfb",
			link: "rgba(255, 255, 255, 0)",
			"link-hover": "rgba(255, 255, 255, 0)",
			"link-active": "#eaf3ff",
			ghost: "rgba(255, 255, 255, 0)",
			"ghost-hover": "rgba(27, 28, 30, 0.06)",
			"ghost-active": "rgba(255, 255, 255, 0)"
		},
		"switch": {
			thumb: "#ffffff",
			"default": "#c5ccd3",
			"default-checked": "#2a8bfb"
		},
		checkbox: {
			"default": "#ffffff",
			checked: "#2a8bfb",
			disabled: "#c5ccd3",
			error: "#ffffff"
		},
		radio: {
			"default": "#ffffff",
			checked: "#ffffff",
			disabled: "#c5ccd3",
			error: "#ffffff"
		},
		input: {
			"outline-default": "#ffffff",
			"outline-hover": "#ffffff",
			"outline-focus": "#ffffff",
			"outline-error": "#ffffff",
			"outline-disabled": "#f1f5f8",
			"filled-default": "#f1f5f8",
			"filled-hover": "#f1f5f8",
			"filled-focus": "#ffffff",
			"filled-error": "#ffffff",
			"filled-disabled": "#f1f5f8",
			"flushed-default": "#ffffff",
			"flushed-hover": "#ffffff",
			"flushed-focus": "#ffffff",
			"flushed-error": "#ffffff",
			"flushed-disabled": "#f1f5f8"
		},
		select: {
			disabled: "#f1f5f8",
			"option-focus": "#eaf3ff",
			"option-active": "#eaf3ff",
			"option-disabled": "#f1f5f8"
		},
		tag: {
			"default": "#002e61",
			subtle: "#eaf3ff"
		},
		toast: {
			success: "#def3e9",
			info: "#eaf3ff",
			error: "#fddcdb",
			warning: "#fdf3c4"
		},
		"input-stepper": {
			"default": "#ffffff",
			hover: "rgba(234, 243, 255, 0.5)",
			active: "#eaf3ff",
			disabled: "#ffffff",
			divider: "#f1f5f8"
		},
		scrollbar: {
			track: "rgba(255, 255, 255, 0)",
			thumb: "#afb7bf"
		},
		spinner: {
			track: "#f1f5f8",
			thumb: "#2a8bfb"
		},
		avatar: {
			"default": "#002e61",
			image: "rgba(255, 255, 255, 0)"
		},
		modal: {
			overlay: "rgba(0, 46, 97, 0.33)"
		},
		tabs: {
			"soft-rounded-active": "#eaf3ff"
		},
		textarea: {
			"default": "#ffffff",
			hover: "#ffffff",
			active: "#ffffff",
			error: "#ffffff",
			disabled: "#f1f5f8"
		},
		tooltip: {
			"default": "#002e61"
		},
		step: {
			"default": "#ffffff",
			hover: "#ffffff",
			active: "#ffffff",
			selected: "#ffffff"
		},
		"flip-button": {
			"default-deselected-blue": "#eaf3ff",
			"hover-deselected-blue": "rgba(234, 243, 255, 0.5)",
			"default-selected-blue": "#2a8bfb",
			"default-deselected-brand": "#f1f5f8",
			"hover-deselected-brand": "rgba(241, 245, 248, 0.5)",
			"default-selected-brand": "#002e61"
		},
		"pane-divider": {
			"default": "#1b1c1e"
		},
		blinker: {
			brand: {
				firstLayer: "#2a8bfb",
				backLayer: "rgba(42, 139, 251, 0.06)"
			},
			success: {
				firstLayer: "#59c28f",
				backLayer: "rgba(89, 194, 143, 0.06)"
			},
			warning: {
				firstLayer: "#fb802d",
				backLayer: "rgba(251, 128, 45, 0.06)"
			},
			pink: {
				firstLayer: "#e3679f",
				backLayer: "rgba(227, 103, 159, 0.06)"
			}
		},
		expandableMenu: {
			"default": "rgba(255, 255, 255, 0)",
			hover: "rgba(234, 243, 255, 0.5)",
			active: "#eaf3ff",
			disabled: "#f1f5f8"
		},
		badge: {
			solid: {
				gray: "#848894",
				mtBlue: "#002e61",
				error: "#f6504b"
			},
			outlined: {
				gray: "#848894",
				mtBlue: "#002e61",
				error: "#f6504b"
			},
			subtle: {
				gray: "#c5ccd3",
				mtBlue: "#bbccdf",
				error: "#fddcdb"
			}
		},
		tableRow: {
			highlighted: "rgba(234, 243, 255, 0.5)",
			selected: "#eaf3ff",
			hover: "rgba(27, 28, 30, 0.06)"
		}
	},
	text: {
		"default": "#1b1c1e",
		inverted: "#ffffff",
		subdued: "#afb7bf",
		brand: "#2a8bfb",
		success: "#59c28f",
		danger: "#f6504b",
		button: {
			"default": "#2a8bfb",
			danger: "#c5403c",
			success: "#479b72",
			link: "#2a8bfb",
			"link-hover": "rgba(42, 139, 251, 0.8)",
			aiSubdued: "#319795",
			brandSubdued: "#2a8bfb"
		},
		input: {
			placeholder: "#afb7bf"
		},
		select: {
			placeholder: "#afb7bf"
		},
		textarea: {
			placeholder: "#afb7bf"
		},
		toast: {
			error: "#1b1c1e"
		},
		tabs: {
			"soft-rounded": "#afb7bf",
			"soft-rounded-active": "#2a8bfb"
		},
		"flip-button": {
			"default": "#1b1c1e",
			inverted: "#ffffff"
		},
		badge: {
			solid: {
				mtblue: "#ffffff",
				red: "#ffffff",
				gray: "#ffffff"
			},
			outlined: {
				mtblue: "#002e61",
				red: "#f6504b",
				gray: "#848894"
			},
			subtle: {
				mtblue: "#001227",
				red: "#62201e",
				gray: "#3b3d44"
			}
		},
		expandableMenu: {
			"default": "#1b1c1e",
			active: "#1b1c1e"
		}
	},
	border: {
		wcag: "#e3679f",
		checkbox: {
			"default": "#c5ccd3",
			"default-hover": "rgba(197, 204, 211, 0.5)",
			checked: "#2a8bfb",
			error: "#f6504b",
			disabled: "#c5ccd3"
		},
		radio: {
			"default": "#c5ccd3",
			"default-hover": "rgba(197, 204, 211, 0.5)",
			checked: "#2a8bfb",
			error: "#f6504b",
			disabled: "#c5ccd3"
		},
		input: {
			"default": "#c5ccd3",
			"default-hover": "rgba(197, 204, 211, 0.5)",
			focus: "#2a8bfb",
			error: "#f6504b",
			disabled: "#c5ccd3",
			readonly: "#c5ccd3"
		},
		select: {
			"default": "#c5ccd3",
			"default-hover": "rgba(197, 204, 211, 0.5)",
			focus: "#2a8bfb",
			error: "#f6504b",
			disabled: "#c5ccd3",
			readonly: "#c5ccd3"
		},
		textarea: {
			"default": "#c5ccd3",
			"default-hover": "rgba(197, 204, 211, 0.5)",
			focus: "#2a8bfb",
			error: "#f6504b",
			disabled: "#c5ccd3",
			readonly: "#c5ccd3"
		},
		scrollbar: {
			thumb: "rgba(255, 255, 255, 0)"
		},
		modal: {
			header: "#c5ccd3"
		},
		step: {
			"default": "#c5ccd3",
			hover: "rgba(197, 204, 211, 0.5)",
			selected: "#2a8bfb"
		},
		"flip-button-group": {
			"default": "#c5ccd3"
		},
		"pane-divider": {
			hover: "#2a8bfb"
		}
	},
	icon: {
		toast: {
			success: "#59c28f",
			error: "#f6504b",
			danger: "#f6504b",
			info: "#2a8bfb",
			warning: "#c8b356",
			ai: "#fae06c"
		},
		"input-stepper": {
			"default": "#2a8bfb",
			disabled: "#c5ccd3",
			readOnly: "#c5ccd3"
		},
		select: {
			"default": "#2a8bfb"
		},
		expandableMenu: {
			subduedDefault: "#afb7bf",
			subduedActive: "#2a8bfb",
			"default": "#2a8bfb"
		}
	}
};
var opacity$1 = {
	button: {
		disabled: "40%"
	},
	"switch": {
		disabled: "40%"
	},
	checkbox: {
		disabled: "40%"
	},
	radio: {
		disabled: "40%"
	},
	input: {
		disabled: "40%"
	},
	select: {
		disabled: "40%"
	},
	textarea: {
		disabled: "40%"
	},
	"flip-button": {
		disabled: "40%"
	},
	expandableMenu: {
		disabled: "40%"
	}
};
var spacing$2 = {
	"padding-inline": {
		input: {
			"default": "0.625rem",
			flushed: "0.25rem"
		},
		textarea: {
			"default": "0.625rem"
		},
		select: {
			"default": "0.625rem"
		},
		tooltip: {
			"default": "0.75rem",
			sm: "0.5rem"
		},
		button: {
			xs: "0.5rem",
			sm: "0.75rem",
			md: "1rem",
			lg: "1.5rem"
		},
		step: {
			"default": "0.5rem",
			text: "0rem"
		},
		"flip-button": {
			xs: "0.5rem",
			sm: "0.75rem",
			md: "1rem",
			lg: "1.5rem",
			gap: "0.5rem"
		},
		"flip-button-group": {
			"default": "0.25rem"
		},
		statusBlock: {
			"default": "0.625rem"
		},
		badge: {
			"default": "0.25rem"
		},
		tag: {
			sm: "0.5rem",
			md: "0.5rem",
			lg: "0.75rem",
			gap: "0.25rem"
		},
		toast: {
			containerDefault: "0.375rem",
			containerGap: "0.75rem",
			contentGap: "0.625rem",
			textGap: "0rem",
			containerLeft: "0.75rem"
		},
		"switch": {
			"default": "0.125rem"
		},
		textArea: {
			"default": "0.625rem"
		},
		expandableMenu: {
			"default": "0.625rem"
		},
		tab: {
			"default": "1rem"
		},
		modal: {
			body: "1.5rem",
			footer: "1.5rem",
			header: "1.5rem"
		}
	},
	paddingY: {
		textarea: {
			"default": "0.375rem"
		},
		tooltip: {
			"default": "0.375rem",
			sm: "0.125rem"
		},
		"flip-button-group": {
			"default": "0.25rem"
		}
	},
	paddingRight: {
		"number-input": {
			field: "5rem",
			stepper: "0.5rem"
		}
	},
	paddingBottom: {
		modal: {
			body: "1rem",
			footer: "1rem",
			header: "1.25rem"
		}
	},
	paddingTop: {
		modal: {
			body: "1rem",
			footer: "0rem",
			header: "1.25rem"
		},
		step: {
			label: "0.5rem"
		}
	}
};
var sizing$2 = {
	avatar: {
		"2xs": "1rem",
		xs: "1.5rem",
		sm: "2rem",
		md: "2.25rem",
		lg: "3rem",
		xl: "4rem",
		"2xl": "6rem"
	},
	button: {
		xs: "1.5rem",
		sm: "2rem",
		md: "2.5rem",
		lg: "3rem"
	},
	icon: {
		"default": "1.25rem",
		xs: "1rem",
		sm: "1.25rem",
		md: "1.25rem",
		lg: "1.5rem"
	},
	blinker: {
		"2xs": "0.5rem",
		xs: "0.75rem",
		sm: "1rem",
		md: "1.25rem",
		lg: "1.5rem"
	},
	"flip-button": {
		xs: "1.5rem",
		sm: "2rem",
		md: "2.5rem",
		lg: "3rem"
	},
	badge: {
		"default": "1.25rem"
	},
	statusBlock: {
		sm: "1.5rem",
		md: "2rem"
	},
	tag: {
		sm: "1.25rem",
		md: "1.5rem",
		lg: "2rem"
	},
	expandableMenu: {
		"default": "3rem"
	},
	tab: {
		"default": "2.5rem"
	},
	modal: {
		xs: "20rem",
		sm: "24rem",
		md: "28rem",
		lg: "32rem",
		xl: "36rem",
		"2xl": "42rem",
		"3xl": "48rem",
		"4xl": "56rem",
		"5xl": "64rem",
		"6xl": "72rem",
		huge: "80rem"
	},
	toast: {
		"default": "24rem"
	},
	checkbox: {
		sm: "0.75rem",
		md: "1rem",
		lg: "1.25rem"
	},
	input: {
		sm: "2rem",
		md: "2.5rem",
		lg: "3rem"
	},
	select: {
		sm: "2rem",
		md: "2.5rem",
		lg: "3rem"
	},
	radio: {
		sm: "0.75rem",
		md: "1rem",
		lg: "1.25rem"
	},
	"switch": {
		thumb: {
			sm: "0.75rem",
			md: "1rem",
			lg: "1.5rem"
		},
		"switch": {
			sm: "1.5rem",
			md: "2rem",
			lg: "3rem"
		}
	}
};
var fontSize$2 = {
	"form-label": {
		"default": "0.875rem"
	}
};
var typography$2 = {
	badge: {
		"default": {
			fontFamily: "Inter",
			fontWeight: 700,
			lineHeight: 14,
			fontSize: 12,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		}
	},
	statusBlock: {
		sm: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 22,
			fontSize: 14,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		},
		md: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		}
	},
	tag: {
		sm: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 16,
			fontSize: 12,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		},
		md: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 22,
			fontSize: 14,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		},
		lg: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		}
	},
	toast: {
		title: {
			fontFamily: "Inter",
			fontWeight: 700,
			lineHeight: 22,
			fontSize: 16,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		description: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 14,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		}
	},
	button: {
		xs: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 16,
			fontSize: 12,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		},
		sm: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 22,
			fontSize: 14,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		},
		md: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		},
		lg: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 28,
			fontSize: 20,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		}
	},
	input: {
		sm: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 14,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		md: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		lg: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		}
	},
	select: {
		sm: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 14,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		md: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		lg: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		}
	},
	textArea: {
		sm: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 14,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		md: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		lg: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		}
	},
	expandableMenu: {
		md: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		}
	},
	steps: {
		label: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 22,
			fontSize: 14,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		},
		description: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 22,
			fontSize: 14,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		}
	},
	tab: {
		md: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		}
	},
	modal: {
		title: {
			fontFamily: "Inter",
			fontWeight: 700,
			lineHeight: 22,
			fontSize: 16,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		}
	},
	tooltip: {
		"default": {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 14,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		}
	}
};
var webappComponents = {
	borderRadius: borderRadius$2,
	borderWidth: borderWidth$1,
	color: color$2,
	opacity: opacity$1,
	spacing: spacing$2,
	sizing: sizing$2,
	fontSize: fontSize$2,
	typography: typography$2
};

var alpha$1 = {
	"hover-full": "cc",
	"hover-soft": 80,
	active: "ff",
	overlay: 55,
	"ghost-hover": 10
};
var st$1 = {
	color: {
		success: "#59C28F",
		"success-alt": "#def3e9",
		destructive: "#F6504B",
		"destructive-alt": "#fddcdb",
		warning: "#FB802D",
		"warning-alt": "#fee6d5",
		info: "#c8b356",
		"info-alt": "#fdf3c4",
		brand: "#2A8BFB",
		"brand-alt": "#eaf3ff",
		base: "#FFFFFF",
		"base-alt": "#f1f5f8",
		inverted: "#1B1C1E",
		"inverted-alt": "#002E61",
		subdued: "#afb7bf",
		"subdued-alt": "#c5ccd3",
		ghost: "#FFFFFF00",
		ai: "#319795",
		"ai-alt": "#E6FFFA",
		bookmark: "#FAE06C",
		bg: {
			base: "#FFFFFF",
			layer: "#f1f5f8",
			overlayer: "#dae1e5",
			filled: "#f1f5f8",
			brand: {
				"default": "#2A8BFB",
				active: "#2A8BFB",
				hover: "#2A8BFBcc"
			},
			secondary: {
				"default": "#59C28F",
				active: "#59C28F",
				hover: "#59C28Fcc"
			},
			tertiary: {
				"default": "#002E61",
				hover: "#002E61cc",
				active: "#002E61"
			},
			success: {
				"default": "#def3e9",
				active: "#def3e9",
				hover: "#def3e9cc"
			},
			error: {
				"default": "#fddcdb",
				active: "#fddcdb",
				hover: "#fddcdbcc"
			},
			warning: {
				"default": "#fee6d5",
				active: "#fee6d5",
				hover: "#fee6d5cc"
			},
			info: {
				"default": "#fdf3c4",
				active: "#fdf3c4",
				hover: "#fdf3c4cc"
			},
			ghost: {
				"default": "#FFFFFF00",
				active: "#FFFFFF00",
				hover: "#FFFFFF00cc"
			},
			ai: {
				"default": "#319795",
				active: "#319795",
				hover: "#319795cc"
			}
		},
		border: {
			"default": "#c5ccd3",
			hover: "#c5ccd380",
			error: "#F6504B",
			warning: "#FB802D",
			info: "#c8b356",
			success: "#59C28F",
			ai: "#319795",
			brand: {
				"default": "#55a2fc",
				hover: "#55a2fc80",
				active: "#55a2fc"
			},
			secondary: {
				"default": "#7acea5",
				hover: "#7acea580",
				active: "#7acea5"
			},
			tertiary: {
				"default": "#2f5581",
				hover: "#2f558180",
				active: "#2f5581"
			}
		},
		text: {
			over: {
				success: "#244e39",
				error: "#62201e",
				warning: "#643312",
				info: "#645a2b",
				brand: "#113864",
				secondary: "#244e39",
				tertiary: "#001227",
				ai: "#285E61"
			},
			link: {
				"default": "#2A8BFB",
				hover: "#2A8BFBcc"
			},
			"over-alt": {
				brand: "#2A8BFB",
				danger: "#c5403c",
				success: "#479b72"
			}
		},
		chart: {
			"1st": "#2A8BFB",
			"2nd": "#7fb9fd",
			"3rd": "#aad1fd",
			"4th": "#7acea5",
			"5th": "#59C28F",
			"6th": "#FAE06C",
			"7th": "#fc9957",
			"8th": "#c96624",
			"9th": "#E3679F",
			"10th": "#e985b2",
			"11th": "#81E6D9",
			"12th": "#4FD1C5",
			"13th": "#fbe689",
			"14th": "#fa9693",
			"15th": "#F6504B",
			"16th": "#c5403c",
			"17th": "#956FAC",
			"18th": "#aa8cbd",
			"19th": "#bfa9cd",
			"20th": "#8ca4c0",
			good: "#7acea5",
			acceptable: "#fbe689",
			poor: "#fc9957",
			bad: "#fa9693"
		}
	},
	borderWidth: {
		xs: "0.0625rem",
		sm: "0.125rem",
		md: "0.25rem",
		lg: "0.375rem",
		xl: "0.5rem"
	},
	borderRadius: {
		xs: "0.125rem",
		sm: "0.25rem",
		md: "0.375rem",
		lg: "0.5rem",
		xl: "1rem",
		full: "999px"
	},
	boxShadow: {
		brand: {
			"default": "0px 0px 10px 0px #2A8BFB80",
			hover: "0px 0px 10px 0px #2A8BFBcc"
		},
		secondary: {
			"default": "0px 0px 10px 0px #59C28F80",
			hover: "0px 0px 10px 0px #59C28Fcc"
		},
		tertiary: {
			"default": "0px 0px 10px 0px #002E6180",
			hover: "0px 0px 10px 0px #002E61cc"
		},
		info: {
			"default": "0px 0px 10px 0px #c8b35680",
			hover: "0px 0px 10px 0px #c8b356cc"
		},
		warning: {
			"default": "0px 0px 10px 0px #FB802D80",
			hover: "0px 0px 10px 0px #FB802Dcc"
		},
		error: {
			"default": "0px 0px 10px 0px #F6504B80",
			hover: "0px 0px 10px 0px #F6504Bcc"
		},
		success: {
			"default": "0px 0px 10px 0px #59C28F80",
			hover: "0px 0px 10px 0px #59C28Fcc"
		}
	}
};
var webapp = {
	alpha: alpha$1,
	st: st$1
};

var borderRadius$1 = {
	button: {
		"default": "0.375rem",
		full: "999px"
	},
	checkbox: {
		"default": "0.125rem",
		full: "999px"
	},
	input: {
		outline: "0.375rem",
		filled: "0.375rem",
		flushed: "0rem"
	},
	select: {
		"default": "0.375rem"
	},
	textarea: {
		"default": "0.375rem"
	},
	tag: {
		"default": "999px"
	},
	toast: {
		"default": "0.375rem"
	},
	"input-stepper": {
		"default": "0.25rem"
	},
	scrollbar: {
		thumb: "0.5rem"
	},
	avatar: {
		rounded: "999px",
		square: "0.375rem"
	},
	modal: {
		dialog: "0.375rem"
	},
	tooltip: {
		"default": "0.25rem"
	},
	"flip-button": {
		"default": "0.375rem"
	},
	badge: {
		"default": "0.125rem"
	},
	statusBlock: {
		"default": "0.375rem"
	},
	"switch": {
		full: "999px"
	}
};
var borderWidth = {
	radio: {
		"default": "0.0625rem",
		"checked-sm": "0.25rem",
		"checked-md": "0.25rem",
		"checked-lg": "0.375rem"
	},
	input: {
		"default": "0.0625rem"
	},
	select: {
		"default": "0.0625rem"
	},
	textarea: {
		"default": "0.0625rem"
	},
	spinner: {
		"default": "0.125rem"
	},
	modal: {
		header: "0.0625rem"
	},
	step: {
		"default": "0.125rem"
	},
	"flip-button-group": {
		"default": "0.125rem"
	},
	badge: {
		outlined: "0.0625rem"
	}
};
var color$1 = {
	background: {
		"default": "#242529",
		button: {
			"default": "#3b3d44",
			"default-hover": "rgba(59, 61, 68, 0.5)",
			"default-active": "#3b3d44",
			danger: "#f6504b",
			"danger-hover": "rgba(246, 80, 75, 0.5)",
			"danger-active": "#f6504b",
			success: "#479b72",
			"success-hover": "rgba(71, 155, 114, 0.5)",
			"success-active": "#479b72",
			brand: "#55a2fc",
			"brand-hover": "rgba(85, 162, 252, 0.8)",
			"brand-active": "#55a2fc",
			link: "rgba(255, 255, 255, 0)",
			"link-hover": "rgba(255, 255, 255, 0)",
			"link-active": "#3b3d44",
			ghost: "rgba(255, 255, 255, 0)",
			"ghost-hover": "rgba(255, 255, 255, 0.06)",
			"ghost-active": "rgba(255, 255, 255, 0)"
		},
		"switch": {
			thumb: "#242529",
			"default": "#c5ccd3",
			"default-checked": "#55a2fc"
		},
		checkbox: {
			"default": "#242529",
			checked: "#55a2fc",
			disabled: "#c5ccd3",
			error: "#242529"
		},
		radio: {
			"default": "#242529",
			checked: "#242529",
			disabled: "#c5ccd3",
			error: "#242529"
		},
		input: {
			"outline-default": "#242529",
			"outline-hover": "#242529",
			"outline-focus": "#242529",
			"outline-error": "#242529",
			"outline-disabled": "#3b3d44",
			"filled-default": "#3b3d44",
			"filled-hover": "#3b3d44",
			"filled-focus": "#242529",
			"filled-error": "#242529",
			"filled-disabled": "#3b3d44",
			"flushed-default": "#242529",
			"flushed-hover": "#242529",
			"flushed-focus": "#242529",
			"flushed-error": "#242529",
			"flushed-disabled": "#3b3d44"
		},
		select: {
			disabled: "#3b3d44",
			"option-focus": "#3b3d44",
			"option-active": "#3b3d44",
			"option-disabled": "#3b3d44"
		},
		tag: {
			"default": "#bbccdf",
			subtle: "#3b3d44"
		},
		toast: {
			success: "#479b72",
			info: "#3b3d44",
			error: "#f6504b",
			warning: "#968641"
		},
		"input-stepper": {
			"default": "#242529",
			hover: "rgba(59, 61, 68, 0.5)",
			active: "#3b3d44",
			disabled: "#242529",
			divider: "#3b3d44"
		},
		scrollbar: {
			track: "rgba(255, 255, 255, 0)",
			thumb: "#afb7bf"
		},
		spinner: {
			track: "#3b3d44",
			thumb: "#55a2fc"
		},
		avatar: {
			"default": "#bbccdf",
			image: "rgba(255, 255, 255, 0)"
		},
		modal: {
			overlay: "rgba(187, 204, 223, 0.33)"
		},
		tabs: {
			"soft-rounded-active": "#3b3d44"
		},
		textarea: {
			"default": "#242529",
			hover: "#242529",
			active: "#242529",
			error: "#242529",
			disabled: "#3b3d44"
		},
		tooltip: {
			"default": "#bbccdf"
		},
		step: {
			"default": "#242529",
			hover: "#242529",
			active: "#242529",
			selected: "#242529"
		},
		"flip-button": {
			"default-deselected-blue": "#3b3d44",
			"hover-deselected-blue": "rgba(59, 61, 68, 0.5)",
			"default-selected-blue": "#55a2fc",
			"default-deselected-brand": "#3b3d44",
			"hover-deselected-brand": "rgba(59, 61, 68, 0.5)",
			"default-selected-brand": "#bbccdf"
		},
		"pane-divider": {
			"default": "#ffffff"
		},
		blinker: {
			brand: {
				firstLayer: "#55a2fc",
				backLayer: "rgba(85, 162, 252, 0.06)"
			},
			success: {
				firstLayer: "#bde7d2",
				backLayer: "rgba(189, 231, 210, 0.06)"
			},
			warning: {
				firstLayer: "#fdccab",
				backLayer: "rgba(253, 204, 171, 0.06)"
			},
			pink: {
				firstLayer: "#e3679f",
				backLayer: "rgba(227, 103, 159, 0.06)"
			}
		},
		expandableMenu: {
			"default": "rgba(255, 255, 255, 0)",
			hover: "rgba(59, 61, 68, 0.5)",
			active: "#3b3d44",
			disabled: "#3b3d44"
		},
		badge: {
			solid: {
				gray: "#848894",
				mtBlue: "#bbccdf",
				error: "#fbb9b7"
			},
			outlined: {
				gray: "#848894",
				mtBlue: "#bbccdf",
				error: "#fbb9b7"
			},
			subtle: {
				gray: "#c5ccd3",
				mtBlue: "#bbccdf",
				error: "#f6504b"
			}
		},
		tableRow: {
			highlighted: "rgba(59, 61, 68, 0.5)",
			selected: "#3b3d44",
			hover: "rgba(255, 255, 255, 0.06)"
		}
	},
	text: {
		"default": "#ffffff",
		inverted: "#242529",
		subdued: "#afb7bf",
		brand: "#55a2fc",
		success: "#bde7d2",
		danger: "#fbb9b7",
		button: {
			"default": "#ffffff",
			danger: "#1b1c1e",
			success: "#1b1c1e",
			link: "#55a2fc",
			"link-hover": "rgba(85, 162, 252, 0.8)",
			aiSubdued: "#38b2ac",
			brandSubdued: "#55a2fc"
		},
		input: {
			placeholder: "#afb7bf"
		},
		select: {
			placeholder: "#afb7bf"
		},
		textarea: {
			placeholder: "#afb7bf"
		},
		toast: {
			error: "#ffffff"
		},
		tabs: {
			"soft-rounded": "#afb7bf",
			"soft-rounded-active": "#55a2fc"
		},
		"flip-button": {
			"default": "#ffffff",
			inverted: "#242529"
		},
		badge: {
			solid: {
				mtblue: "#242529",
				red: "#242529",
				gray: "#ffffff"
			},
			outlined: {
				mtblue: "#bbccdf",
				red: "#fbb9b7",
				gray: "#848894"
			},
			subtle: {
				mtblue: "#001227",
				red: "#62201e",
				gray: "#3b3d44"
			}
		},
		expandableMenu: {
			"default": "#ffffff",
			active: "#ffffff"
		}
	},
	border: {
		wcag: "#e3679f",
		checkbox: {
			"default": "#c5ccd3",
			"default-hover": "rgba(197, 204, 211, 0.5)",
			checked: "#55a2fc",
			error: "#fbb9b7",
			disabled: "#c5ccd3"
		},
		radio: {
			"default": "#c5ccd3",
			"default-hover": "rgba(197, 204, 211, 0.5)",
			checked: "#55a2fc",
			error: "#fbb9b7",
			disabled: "#c5ccd3"
		},
		input: {
			"default": "#c5ccd3",
			"default-hover": "rgba(197, 204, 211, 0.5)",
			focus: "#55a2fc",
			error: "#fbb9b7",
			disabled: "#c5ccd3",
			readonly: "#c5ccd3"
		},
		select: {
			"default": "#c5ccd3",
			"default-hover": "rgba(197, 204, 211, 0.5)",
			focus: "#55a2fc",
			error: "#fbb9b7",
			disabled: "#c5ccd3",
			readonly: "#c5ccd3"
		},
		textarea: {
			"default": "#c5ccd3",
			"default-hover": "rgba(197, 204, 211, 0.5)",
			focus: "#55a2fc",
			error: "#fbb9b7",
			disabled: "#c5ccd3",
			readonly: "#c5ccd3"
		},
		scrollbar: {
			thumb: "rgba(255, 255, 255, 0)"
		},
		modal: {
			header: "#c5ccd3"
		},
		step: {
			"default": "#c5ccd3",
			hover: "rgba(197, 204, 211, 0.5)",
			selected: "#55a2fc"
		},
		"flip-button-group": {
			"default": "#c5ccd3"
		},
		"pane-divider": {
			hover: "#55a2fc"
		}
	},
	icon: {
		toast: {
			success: "#bde7d2",
			error: "#fbb9b7",
			danger: "#fbb9b7",
			info: "#55a2fc",
			warning: "#fbe689",
			ai: "#fae06c"
		},
		"input-stepper": {
			"default": "#55a2fc",
			disabled: "#c5ccd3",
			readOnly: "#c5ccd3"
		},
		select: {
			"default": "#55a2fc"
		},
		expandableMenu: {
			subduedDefault: "#afb7bf",
			subduedActive: "#55a2fc",
			"default": "#55a2fc"
		}
	}
};
var opacity = {
	button: {
		disabled: "40%"
	},
	"switch": {
		disabled: "40%"
	},
	checkbox: {
		disabled: "40%"
	},
	radio: {
		disabled: "40%"
	},
	input: {
		disabled: "40%"
	},
	select: {
		disabled: "40%"
	},
	textarea: {
		disabled: "40%"
	},
	"flip-button": {
		disabled: "40%"
	},
	expandableMenu: {
		disabled: "40%"
	}
};
var spacing$1 = {
	"padding-inline": {
		input: {
			"default": "0.625rem",
			flushed: "0.25rem"
		},
		textarea: {
			"default": "0.625rem"
		},
		select: {
			"default": "0.625rem"
		},
		tooltip: {
			"default": "0.75rem",
			sm: "0.5rem"
		},
		button: {
			xs: "0.5rem",
			sm: "0.75rem",
			md: "1rem",
			lg: "1.5rem"
		},
		step: {
			"default": "0.5rem",
			text: "0rem"
		},
		"flip-button": {
			xs: "0.5rem",
			sm: "0.75rem",
			md: "1rem",
			lg: "1.5rem",
			gap: "0.5rem"
		},
		"flip-button-group": {
			"default": "0.25rem"
		},
		statusBlock: {
			"default": "0.625rem"
		},
		badge: {
			"default": "0.25rem"
		},
		tag: {
			sm: "0.5rem",
			md: "0.5rem",
			lg: "0.75rem",
			gap: "0.25rem"
		},
		toast: {
			containerDefault: "0.375rem",
			containerGap: "0.75rem",
			contentGap: "0.625rem",
			textGap: "0rem",
			containerLeft: "0.75rem"
		},
		"switch": {
			"default": "0.125rem"
		},
		textArea: {
			"default": "0.625rem"
		},
		expandableMenu: {
			"default": "0.625rem"
		},
		tab: {
			"default": "1rem"
		},
		modal: {
			body: "1.5rem",
			footer: "1.5rem",
			header: "1.5rem"
		}
	},
	paddingY: {
		textarea: {
			"default": "0.375rem"
		},
		tooltip: {
			"default": "0.375rem",
			sm: "0.125rem"
		},
		"flip-button-group": {
			"default": "0.25rem"
		}
	},
	paddingRight: {
		"number-input": {
			field: "5rem",
			stepper: "0.5rem"
		}
	},
	paddingBottom: {
		modal: {
			body: "1rem",
			footer: "1rem",
			header: "1.25rem"
		}
	},
	paddingTop: {
		modal: {
			body: "1rem",
			footer: "0rem",
			header: "1.25rem"
		},
		step: {
			label: "0.5rem"
		}
	}
};
var sizing$1 = {
	avatar: {
		"2xs": "1rem",
		xs: "1.5rem",
		sm: "2rem",
		md: "2.25rem",
		lg: "3rem",
		xl: "4rem",
		"2xl": "6rem"
	},
	button: {
		xs: "1.5rem",
		sm: "2rem",
		md: "2.5rem",
		lg: "3rem"
	},
	icon: {
		"default": "1.25rem",
		xs: "1rem",
		sm: "1.25rem",
		md: "1.25rem",
		lg: "1.5rem"
	},
	blinker: {
		"2xs": "0.5rem",
		xs: "0.75rem",
		sm: "1rem",
		md: "1.25rem",
		lg: "1.5rem"
	},
	"flip-button": {
		xs: "1.5rem",
		sm: "2rem",
		md: "2.5rem",
		lg: "3rem"
	},
	badge: {
		"default": "1.25rem"
	},
	statusBlock: {
		sm: "1.5rem",
		md: "2rem"
	},
	tag: {
		sm: "1.25rem",
		md: "1.5rem",
		lg: "2rem"
	},
	expandableMenu: {
		"default": "3rem"
	},
	tab: {
		"default": "2.5rem"
	},
	modal: {
		xs: "20rem",
		sm: "24rem",
		md: "28rem",
		lg: "32rem",
		xl: "36rem",
		"2xl": "42rem",
		"3xl": "48rem",
		"4xl": "56rem",
		"5xl": "64rem",
		"6xl": "72rem",
		huge: "80rem"
	},
	toast: {
		"default": "24rem"
	},
	checkbox: {
		sm: "0.75rem",
		md: "1rem",
		lg: "1.25rem"
	},
	input: {
		sm: "2rem",
		md: "2.5rem",
		lg: "3rem"
	},
	select: {
		sm: "2rem",
		md: "2.5rem",
		lg: "3rem"
	},
	radio: {
		sm: "0.75rem",
		md: "1rem",
		lg: "1.25rem"
	},
	"switch": {
		thumb: {
			sm: "0.75rem",
			md: "1rem",
			lg: "1.5rem"
		},
		"switch": {
			sm: "1.5rem",
			md: "2rem",
			lg: "3rem"
		}
	}
};
var fontSize$1 = {
	"form-label": {
		"default": "0.875rem"
	}
};
var typography$1 = {
	badge: {
		"default": {
			fontFamily: "Inter",
			fontWeight: 700,
			lineHeight: 14,
			fontSize: 12,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		}
	},
	statusBlock: {
		sm: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 22,
			fontSize: 14,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		},
		md: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		}
	},
	tag: {
		sm: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 16,
			fontSize: 12,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		},
		md: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 22,
			fontSize: 14,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		},
		lg: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		}
	},
	toast: {
		title: {
			fontFamily: "Inter",
			fontWeight: 700,
			lineHeight: 22,
			fontSize: 16,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		description: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 14,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		}
	},
	button: {
		xs: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 16,
			fontSize: 12,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		},
		sm: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 22,
			fontSize: 14,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		},
		md: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		},
		lg: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 28,
			fontSize: 20,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		}
	},
	input: {
		sm: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 14,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		md: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		lg: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		}
	},
	select: {
		sm: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 14,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		md: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		lg: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		}
	},
	textArea: {
		sm: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 14,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		md: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		lg: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		}
	},
	expandableMenu: {
		md: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		}
	},
	steps: {
		label: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 22,
			fontSize: 14,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		},
		description: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 22,
			fontSize: 14,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		}
	},
	tab: {
		md: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: 24,
			fontSize: 16,
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		}
	},
	modal: {
		title: {
			fontFamily: "Inter",
			fontWeight: 700,
			lineHeight: 22,
			fontSize: 16,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		}
	},
	tooltip: {
		"default": {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: 24,
			fontSize: 14,
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		}
	}
};
var tottComponents = {
	borderRadius: borderRadius$1,
	borderWidth: borderWidth,
	color: color$1,
	opacity: opacity,
	spacing: spacing$1,
	sizing: sizing$1,
	fontSize: fontSize$1,
	typography: typography$1
};

var alpha = {
	"hover-full": "cc",
	"hover-soft": 80,
	active: "ff",
	overlay: 55,
	"ghost-hover": 10
};
var st = {
	color: {
		success: "#bde7d2",
		"success-alt": "#479b72",
		destructive: "#fbb9b7",
		"destructive-alt": "#F6504B",
		warning: "#fdccab",
		"warning-alt": "#FB802D",
		info: "#fbe689",
		"info-alt": "#968641",
		brand: "#55a2fc",
		"brand-alt": "#3b3d44",
		base: "#242529",
		"base-alt": "#3b3d44",
		inverted: "#FFFFFF",
		"inverted-alt": "#bbccdf",
		subdued: "#afb7bf",
		"subdued-alt": "#c5ccd3",
		ai: "#38B2AC",
		"ai-alt": "#2C7A7B",
		bookmark: "#FAE06C",
		ghost: "#FFFFFF00",
		bg: {
			base: "#242529",
			layer: "#3b3d44",
			overlayer: "#535660",
			filled: "#3b3d44",
			brand: {
				"default": "#55a2fc",
				active: "#55a2fc",
				hover: "#55a2fccc"
			},
			secondary: {
				"default": "#7acea5",
				active: "#7acea5",
				hover: "#7acea5cc"
			},
			tertiary: {
				"default": "#2f5581",
				active: "#2f5581",
				hover: "#2f5581cc"
			},
			success: {
				"default": "#479b72",
				active: "#479b72",
				hover: "#479b72cc"
			},
			error: {
				"default": "#F6504B",
				active: "#F6504B",
				hover: "#F6504Bcc"
			},
			warning: {
				"default": "#FB802D",
				active: "#FB802D",
				hover: "#FB802Dcc"
			},
			info: {
				"default": "#968641",
				active: "#968641",
				hover: "#968641cc"
			},
			ghost: {
				"default": "#FFFFFF00",
				hover: "#FFFFFF00cc",
				active: "#FFFFFF00"
			},
			ai: {
				"default": "#38B2AC",
				active: "#38B2AC",
				hover: "#38B2ACcc"
			}
		},
		border: {
			"default": "#FFFFFF",
			hover: "#f1f5f8",
			error: "#fbb9b7",
			warning: "#fdccab",
			info: "#fbe689",
			success: "#bde7d2",
			ai: "#319795",
			brand: {
				"default": "#226fc9",
				hover: "#226fc9cc",
				active: "#226fc9"
			},
			secondary: {
				"default": "#479b72",
				hover: "#479b72cc",
				active: "#479b72"
			},
			tertiary: {
				"default": "#00254e",
				hover: "#00254ecc",
				active: "#00254e"
			}
		},
		text: {
			over: {
				success: "#eef9f4",
				error: "#feeeed",
				warning: "#fff2ea",
				info: "#fffcf0",
				brand: "#eaf3ff",
				secondary: "#eef9f4",
				tertiary: "#eaf3ff",
				ai: "#E6FFFA"
			},
			link: {
				"default": "#55a2fc",
				hover: "#55a2fccc"
			},
			"over-alt": {
				brand: "#FFFFFF",
				danger: "#1B1C1E",
				success: "#1B1C1E"
			}
		},
		chart: {
			"1st": "#2A8BFB",
			"2nd": "#7fb9fd",
			"3rd": "#aad1fd",
			"4th": "#7acea5",
			"5th": "#59C28F",
			"6th": "#FAE06C",
			"7th": "#fc9957",
			"8th": "#c96624",
			"9th": "#E3679F",
			"10th": "#e985b2",
			"11th": "#81E6D9",
			"12th": "#4FD1C5",
			"13th": "#fbe689",
			"14th": "#fa9693",
			"15th": "#F6504B",
			"16th": "#c5403c",
			"17th": "#956FAC",
			"18th": "#aa8cbd",
			"19th": "#bfa9cd",
			"20th": "#8ca4c0",
			good: "#7acea5",
			acceptable: "#fbe689",
			poor: "#fc9957",
			bad: "#fa9693"
		}
	},
	borderWidth: {
		xs: "0.0625rem",
		sm: "0.125rem",
		md: "0.25rem",
		lg: "0.375rem",
		xl: "0.5rem"
	},
	borderRadius: {
		xs: "0.125rem",
		sm: "0.25rem",
		md: "0.375rem",
		lg: "0.5rem",
		xl: "1rem",
		full: "999px"
	},
	boxShadow: {
		brand: {
			"default": "0px 0px 10px 0px #55a2fc80",
			hover: "0px 0px 10px 0px #55a2fccc"
		},
		secondary: {
			"default": "0px 0px 10px 0px #7acea580",
			hover: "0px 0px 10px 0px #7acea5cc"
		},
		tertiary: {
			"default": "0px 10px 0px 0px #2f558180",
			hover: "0px 0px 10px 0px #2f5581cc"
		},
		info: {
			"default": "0px 0px 10px 0px #fbe68980",
			hover: "0px 0px 10px 0px #fbe689cc"
		},
		warning: {
			"default": "0px 0px 10px 0px #fdccab80",
			hover: "0px 0px 10px 0px #fdccabcc"
		},
		success: {
			"default": "0px 0px 10px 0px #bde7d280",
			hover: "0px 0px 10px 0px #bde7d2cc"
		},
		error: {
			"default": "0px 0px 10px 0px #fbb9b780",
			hover: "0px 0px 10px 0px #fbb9b7cc"
		}
	}
};
var tott = {
	alpha: alpha,
	st: st
};

var borderRadius = {
	"0": "0rem",
	"2": "0.125rem",
	"4": "0.25rem",
	"6": "0.375rem",
	"8": "0.5rem",
	"16": "1rem",
	"999": "999px"
};
var color = {
	mono: {
		white: "#ffffff",
		black: "#1b1c1e",
		transparent: "rgba(255, 255, 255, 0)"
	},
	blue: {
		"50": "#eaf3ff",
		"100": "#d4e8fe",
		"200": "#aad1fd",
		"300": "#7fb9fd",
		"400": "#55a2fc",
		"500": "#2a8bfb",
		"600": "#226fc9",
		"700": "#195397",
		"800": "#113864",
		"900": "#081c32"
	},
	gray: {
		"50": "#f1f5f8",
		"100": "#dae1e5",
		"200": "#c5ccd3",
		"300": "#afb7bf",
		"400": "#98a1ac",
		"500": "#848894",
		"600": "#6b6f7b",
		"700": "#535660",
		"800": "#3b3d44",
		"900": "#242529"
	},
	green: {
		"50": "#eef9f4",
		"100": "#def3e9",
		"200": "#bde7d2",
		"300": "#9bdabc",
		"400": "#7acea5",
		"500": "#59c28f",
		"600": "#479b72",
		"700": "#357456",
		"800": "#244e39",
		"900": "#09130e"
	},
	mediatoolBlue: {
		"50": "#eaf3ff",
		"100": "#bbccdf",
		"200": "#8ca4c0",
		"300": "#5e7da0",
		"400": "#2f5581",
		"500": "#002e61",
		"600": "#00254e",
		"700": "#001c3a",
		"800": "#001227",
		"900": "#000913"
	},
	pink: {
		"50": "#fcf0f5",
		"100": "#f9e1ec",
		"200": "#f4c2d9",
		"300": "#eea4c5",
		"400": "#e985b2",
		"500": "#e3679f",
		"600": "#b6527f",
		"700": "#883e5f",
		"800": "#5b2940",
		"900": "#2d1520"
	},
	red: {
		"50": "#feeeed",
		"100": "#fddcdb",
		"200": "#fbb9b7",
		"300": "#fa9693",
		"400": "#f8736f",
		"500": "#f6504b",
		"600": "#c5403c",
		"700": "#94302d",
		"800": "#62201e",
		"900": "#31100f"
	},
	orange: {
		"50": "#fff2ea",
		"100": "#fee6d5",
		"200": "#fdccab",
		"300": "#fdb381",
		"400": "#fc9957",
		"500": "#fb802d",
		"600": "#c96624",
		"700": "#974d1b",
		"800": "#643312",
		"900": "#321a09"
	},
	purple: {
		"50": "#f4f1f7",
		"100": "#eae2ee",
		"200": "#d5c5de",
		"300": "#bfa9cd",
		"400": "#aa8cbd",
		"500": "#956fac",
		"600": "#77598a",
		"700": "#594367",
		"800": "#3c2c45",
		"900": "#1e1622"
	},
	yellow: {
		"50": "#fffcf0",
		"100": "#fef9e2",
		"200": "#fdf3c4",
		"300": "#fceca7",
		"400": "#fbe689",
		"500": "#fae06c",
		"600": "#c8b356",
		"700": "#968641",
		"800": "#645a2b",
		"900": "#322d16"
	},
	teal: {
		"50": "#e6fffa",
		"100": "#b2f5ea",
		"200": "#81e6d9",
		"300": "#4fd1c5",
		"400": "#38b2ac",
		"500": "#319795",
		"600": "#2c7a7b",
		"700": "#285e61",
		"800": "#234e52",
		"900": "#1d4044"
	}
};
var spacing = {
	"0": "0rem",
	"1": "0.25rem",
	"2": "0.5rem",
	"3": "0.75rem",
	"4": "1rem",
	"5": "1.25rem",
	"6": "1.5rem",
	"7": "1.75rem",
	"8": "2rem",
	"9": "2.25rem",
	"10": "2.5rem",
	"12": "3rem",
	"14": "3.5rem",
	"16": "4rem",
	"20": "5rem",
	"24": "6rem",
	"28": "7rem",
	"32": "8rem",
	"36": "9rem",
	"40": "10rem",
	"44": "11rem",
	"48": "12rem",
	"52": "13rem",
	"56": "14rem",
	"60": "15rem",
	"64": "16rem",
	"72": "18rem",
	"80": "20rem",
	"96": "24rem",
	"0a": "0.125rem",
	"1a": "0.375rem",
	"2a": "0.625rem",
	"3a": "0.875rem",
	px: "0.0625rem"
};
var sizing = {
	"0": "0rem",
	"1": "0.25rem",
	"2": "0.5rem",
	"3": "0.75rem",
	"4": "1rem",
	"5": "1.25rem",
	"6": "1.5rem",
	"7": "1.75rem",
	"8": "2rem",
	"9": "2.25rem",
	"10": "2.5rem",
	"12": "3rem",
	"14": "3.5rem",
	"16": "4rem",
	"20": "5rem",
	"24": "6rem",
	"28": "7rem",
	"32": "8rem",
	"36": "9rem",
	"40": "10rem",
	"44": "11rem",
	"48": "12rem",
	"52": "13rem",
	"56": "14rem",
	"60": "15rem",
	"64": "16rem",
	"72": "18rem",
	"80": "20rem",
	"96": "24rem",
	"0a": "0.125rem",
	"1a": "0.375rem",
	"2a": "0.625rem",
	"3a": "0.875rem",
	px: "0.0625rem",
	"3xs": "14rem",
	"2xs": "16rem",
	xs: "20rem",
	sm: "24rem",
	md: "28rem",
	lg: "32rem",
	xl: "36rem",
	"2xl": "42rem",
	"3xl": "48rem",
	"4xl": "56rem",
	"5xl": "64rem",
	"6xl": "72rem",
	"7xl": "80rem",
	"8xl": "90rem",
	container: {
		sm: "40rem",
		md: "48rem",
		lg: "64rem",
		xl: "80rem"
	}
};
var lineHeights = {
	"1": "0.875rem",
	"2": "1rem",
	"3": "1.375rem",
	"4": "1.5rem",
	"5": "1.75rem",
	"6": "2rem",
	"7": "2.375rem",
	"8": "2.75rem",
	"9": "3.375rem",
	auto: "AUTO"
};
var fontWeights = {
	bold: 700,
	regular: 400,
	semiBold: 600,
	medium: 500
};
var fontSize = {
	tiny: "0.625rem",
	xs: "0.75rem",
	sm: "0.875rem",
	md: "1rem",
	lg: "1.25rem",
	xl: "1.5rem",
	"2xl": "1.75rem",
	"3xl": "2rem",
	"4xl": "2.5rem"
};
var boxShadow = {
	xs: "0px 0px 0px 1px #0000000d",
	sm: "0px 1px 2px 0px #0000000d",
	base: "0px 1px 2px 0px #0000000f, 0px 1px 3px 0px #0000001a",
	md: "0px 2px 4px -1px #0000000f, 0px 4px 6px -1px #0000001a",
	lg: "0px 4px 6px -2px #0000000d, 0px 10px 15px -3px #0000001a",
	xl: "0px 10px 10px -5px #0000000a, 0px 20px 25px -5px #0000001a",
	"2xl": "0px 25px 50px -12px #00000040",
	outline: "0px 0px 0px 3px #3f99e1b3",
	inner: "inset 0px 2px 4px 0px #0000000f",
	"kbd-inner": "inset 0px -2px 0px 0px #e2e8f0",
	"dark-lg": "0px 15px 40px 0px #00000066, 0px 5px 10px 0px #00000033, 0px 0px 0px 1px #0000001a"
};
var zIndex = {
	hide: -999,
	base: 1,
	docked: 100,
	dropdown: 125,
	sticky: 250,
	banner: 500,
	overlay: 1000,
	modal: 1500,
	popover: 2000,
	toast: 2500,
	tooltip: 3000,
	maxnus: 999999
};
var typography = {
	headings: {
		h1: {
			fontFamily: "Inter",
			fontWeight: 700,
			lineHeight: "2.375rem",
			fontSize: "1.75rem",
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		h2: {
			fontFamily: "Inter",
			fontWeight: 700,
			lineHeight: "2rem",
			fontSize: "1.5rem",
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		h3: {
			fontFamily: "Inter",
			fontWeight: 700,
			lineHeight: "1.75rem",
			fontSize: "1.25rem",
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		h4: {
			fontFamily: "Inter",
			fontWeight: 700,
			lineHeight: "1.375rem",
			fontSize: "1rem",
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		h5: {
			fontFamily: "Inter",
			fontWeight: 700,
			lineHeight: "1rem",
			fontSize: "0.875rem",
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		h6: {
			fontFamily: "Inter",
			fontWeight: 700,
			lineHeight: "0.875rem",
			fontSize: "0.75rem",
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		}
	},
	body: {
		body16: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: "1.5rem",
			fontSize: "1rem",
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		body14: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: "1.5rem",
			fontSize: "0.875rem",
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		}
	},
	stylized: {
		blockquote: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: "1.75rem",
			fontSize: "1.25rem",
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		CAPITALIZED: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: "AUTO",
			fontSize: "0.875rem",
			letterSpacing: 2,
			textCase: "uppercase",
			textDecoration: "none"
		},
		small: {
			fontFamily: "Inter",
			fontWeight: 400,
			lineHeight: "AUTO",
			fontSize: "0.75rem",
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		tiny: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: "AUTO",
			fontSize: "0.625rem",
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		},
		lead: {
			fontFamily: "Inter",
			fontWeight: 700,
			lineHeight: "1.5rem",
			fontSize: "0.875rem",
			letterSpacing: 0,
			textCase: "none",
			textDecoration: "none"
		}
	},
	labels: {
		tinyLabel: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: "0.875rem",
			fontSize: "0.625rem",
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		},
		xsLabel: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: "1rem",
			fontSize: "0.75rem",
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		},
		smLabel: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: "1.375rem",
			fontSize: "0.875rem",
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		},
		mdLabel: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: "1.5rem",
			fontSize: "1rem",
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		},
		lgLabel: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: "1.75rem",
			fontSize: "1.25rem",
			letterSpacing: 0.2,
			textCase: "none",
			textDecoration: "none"
		}
	},
	numVal: {
		tinyMono: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: "0.875rem",
			fontSize: "0.625rem",
			letterSpacing: 1.2,
			textCase: "none",
			textDecoration: "none"
		},
		xsMono: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: "1rem",
			fontSize: "0.75rem",
			letterSpacing: 1.2,
			textCase: "none",
			textDecoration: "none"
		},
		smMono: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: "1.375rem",
			fontSize: "0.875rem",
			letterSpacing: 1.2,
			textCase: "none",
			textDecoration: "none"
		},
		mdMono: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: "1.5rem",
			fontSize: "1rem",
			letterSpacing: 1.2,
			textCase: "none",
			textDecoration: "none"
		},
		lgMono: {
			fontFamily: "Inter",
			fontWeight: 600,
			lineHeight: "1.75rem",
			fontSize: "1.25rem",
			letterSpacing: 1.2,
			textCase: "none",
			textDecoration: "none"
		}
	}
};

exports.TottComponentsTokensSet = tottComponents;
exports.TottSystemTokensSet = tott;
exports.WebappComponentsTokensSet = webappComponents;
exports.WebappSystemTokensSet = webapp;
exports.coreBorderRadius = borderRadius;
exports.coreBoxShadow = boxShadow;
exports.coreFontSize = fontSize;
exports.coreFontWeight = fontWeights;
exports.coreLineHeight = lineHeights;
exports.coreSizing = sizing;
exports.coreSpacing = spacing;
exports.coreZIndex = zIndex;
exports.palette = color;
exports.typography = typography;
